client/my-sites/domains/domain-management/edit/navigation/index.jsx

```diff
@@ -0,0 +1,311 @@
+		const { domain } = this.props;
+		const { expired, pendingTransfer } = domain;
+		return ! pendingTransfer && ! expired;
+	}
+
+	isDomainInGracePeriod() {
+		const { domain, moment } = this.props;
+		const { expiry } = domain;
+		return moment().subtract( 18, 'days' ) <= moment( expiry );
+	}
+
+	getEmail() {
+		const { selectedSite, translate, domain } = this.props;
+
+		if ( ! this.isDomainInNormalState() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ emailManagement( selectedSite.slug, domain.name ) }
+				materialIcon="email"
+				text={ translate( 'Email' ) }
+				description={ null && translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getNameServers() {
+		const { selectedSite, translate, domain } = this.props;
+
+		if ( ! this.isDomainInNormalState() && ! this.isDomainInGracePeriod() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementNameServers( selectedSite.slug, domain.name ) }
+				materialIcon="language"
+				text={ translate( 'Name servers and DNS' ) }
+				description={ null && translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getDnsRecords() {
+		const { selectedSite, translate, domain } = this.props;
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementDns( selectedSite.slug, domain.name ) }
+				materialIcon="language"
+				text={ translate( 'DNS records' ) }
+				description={ null && translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getContactsAndPrivacy() {
+		const { selectedSite, translate, domain } = this.props;
+
+		if ( ! this.isDomainInNormalState() && ! this.isDomainInGracePeriod() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementContactsPrivacy( selectedSite.slug, domain.name ) }
+				materialIcon="chrome_reader_mode"
+				text={ translate( 'Contacts and privacy' ) }
+				description={ null && translate( 'Privacy protection: on' ) }
+			/>
+		);
+	}
+
+	getTransferDomain() {
+		const { selectedSite, translate, domain } = this.props;
+		const { expired } = domain;
+
+		if ( expired && ! this.isDomainInGracePeriod() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementTransfer( selectedSite.slug, domain.name ) }
+				materialIcon="cached"
+				text={ translate( 'Transfer domain' ) }
+				description={ null && translate( 'Transfer lock: off' ) }
+			/>
+		);
+	}
+
+	getDomainConnectMapping() {
+		const { selectedSite, translate, domain } = this.props;
+
+		const { supportsDomainConnect, hasWpcomNameservers, pointsToWpcom } = domain;
+
+		if ( ! supportsDomainConnect || hasWpcomNameservers || pointsToWpcom ) {
+			return;
+		}
+
+		const path = domainManagementDomainConnectMapping( selectedSite.slug, domain.name );
+
+		return (
+			<DomainManagementNavigationItem
+				path={ path }
+				materialIcon="cached"
+				text={ translate( 'Connect your domain' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Connect Your Domain' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-status-new-navigation)